conda.recipe/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# load information from setup.cfg/setup.py
+{% set data = load_setup_py_data() %}
+{% set license = data.get('license') %}
+{% set description = data.get('description') %}
+{% set url = data.get('url') %}
+# this will get the version set by environment variable
+{% set version = environ.get('VERSION') %}
+{% set version_number = environ.get('GIT_DESCRIBE_NUMBER', '0') | string %}
+
+
+package:
+  name: shiver
+  version: {{ version_number }}
+
+source:
+  path: ..
+
+build:
+  noarch: python
+  linux-64: python
+  number: {{ version_number }}
+  string: py{{py}}
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+
+requirements:
+  host:
+    - python
+    - versioningit
+
+  build:
+    - setuptools
+    - versioningit
+    - mantidworkbench
```

Comment:
I don't think that your build needs `mantidworkbench`, `qtpy`, or `pytest`. The conda build doesn't invoke the test suite.

Reply:
removed